debugger/src/test/resources/META-INF/plugin.xml

```diff
@@ -1,6 +1,6 @@
 <idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude" allow-bundled-update="true">
     <id>org.rust.debugger</id>
     <xi:include href="/META-INF/rust-core.xml" xpointer="xpointer(/idea-plugin/*)"/>
-    <depends optional="true" config-file="core-debugger-only.xml">com.intellij.modules.clion</depends>
+    <depends optional="true" config-file="debugger-only.xml">com.intellij.modules.clion</depends>
```

Comment:
By the way, could you please explain why `debugger-only.xml` depends on `com.intellij.modules.clion`, and not on something like`com.intellij.modules.cidr`?

Reply:
The idea was to use only one of optional dependencies: `debugger-only.xml` with CLion (as a part of CLion's `clion-only.xml`) or `native-debug-only.xml` with IDEA (note, `native-debug-only.xml` includes `debugger-only.xml`)

Reply:
Now I see, thanks! Maybe we could mention the idea in the comments to make this optional dependencies thing easier to understand at a glance

Reply:
It will be changed in #8709 where debugger module will be separate entity so this confusion will be fixed automatically